airflow/providers/docker/hooks/docker.py

```diff
@@ -63,7 +63,7 @@ def __init__(
         if not version:
             raise AirflowException('No Docker API version provided')
 
-        conn = self.get_connection(docker_conn_id)
+        conn = self.get_connection(docker_conn_id)  # type: ignore
```

Comment:
What's the problem here? That sounds fixable?

Reply:
Interesting but I didn't get any error here.

Reply:
Maybe that was the reason indeed.

Reply:
It was actually from the operator calling the hook, fixed with a check.